automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Collapse.scala

```diff
@@ -16,8 +17,16 @@ case class Collapse[A <: FireCloudView](queryString: QueryString, private val in
 
   def toggle(): Unit = {
     await visible toggleComponent
-    click on toggleComponent
-    awaitReady()
+    val origState = stateOf(toggleComponent)
+    try {
+      click on toggleComponent
+    } catch {
+      // retry click alternative. probably should be used in doClick
+      // https://github.com/seleniumhq/selenium-google-code-issue-archive/issues/2766
+      case e: WebDriverException => new Actions(webDriver).moveToElement(toggleComponent.element.underlying).click().build.perform()
```

Comment:
I agree that click retrying should be in doClick. Is this hacky way of doing it really necessary, or can we just try to `click on whatever` again?

Reply:
a number of people reported that they had success with alternative click. `moveToElement` actually scrolls the target WebElement into the viewport.

Reply:
So would making `toggleComponent` a `Link` and calling `doClick()` on it.